app/Http/Controllers/QuestionController.php

```diff
@@ -70,9 +120,45 @@ public function edit($id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function update(UpdateQuestionRequest $request, Question $question)
     {
-        //
+        if (Gate::denies('update-question', $question)) {
+            abort(403, "Access denied");
+        }
+        $data = $request->validated();
+        $request->validated();
+        $listTags = explode(", ", $request->tags);
+        $listTagsId = [];
+        foreach ($listTags as $tag) {
+            $tagId = Tag::where('title', $tag)->first();
```

Comment:
em dùng whereIn để lấy ra danh sách tag, sau đó mới for nhé.
làm ntn bị n queries rùi